temporian/implementation/numpy/operators/window/simple_moving_average_test.py

```diff
@@ -0,0 +1,92 @@
+# Copyright 2021 Google LLC.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from absl.testing import absltest
+
+import pandas as pd
+import numpy as np
+
+from temporian.core.operators.simple_moving_average import SimpleMovingAverage
+from temporian.implementation.numpy.operators.window.simple_moving_average import (
+    SimpleMovingAverageNumpyImplementation,
+)
+from temporian.implementation.numpy.data.event import NumpyEvent, NumpyFeature
+from temporian.implementation.numpy.data.sampling import NumpySampling
+from temporian.core.data import event as event_lib
+from temporian.core.data import feature as feature_lib
+from temporian.core.data import dtype as dtype_lib
+
+
+class SimpleMovingAverageOperatorTest(absltest.TestCase):
+    def setUp(self):
+        pass
+
+    # TODO: Add more tests (index, sampling, nan, etc).
+
+    def test_no_index(self):

```

Comment:
Let's try to reuse `PandasSimpleMovingAverageOperator`'s tests which cover several cases (though I agree with trying to make them as small as possible to avoid having data in separate files)

Reply:
I agree we want to reuse them (added details in todo in test file).

As you can see, the test is still too verbosity. We need to create the "pd_to_event" function and use it in the tests (see toso). With "pd_to_event", the pandas tests will be usable directly.
However, creating this "pd_to_event" function should probably be a PR on its own.

Do you you think about creating "pd_to_event" and moving the tests in two separate following PRs?